src/const.h

```diff
@@ -180,31 +190,42 @@ enum SpeakClasses : uint8_t {
 	TALKTYPE_CHANNEL_R1 = 14, //red - #c text
 	TALKTYPE_PRIVATE_RED_FROM = 15, //@name@text
 	TALKTYPE_PRIVATE_RED_TO = 16, //@name@text
-	TALKTYPE_MONSTER_SAY = 35,
-	TALKTYPE_MONSTER_YELL = 36,
+	TALKTYPE_MONSTER_SAY = 36,
```

Comment:
that will debug so many scripts that don't use constants instead of variables :/